src/components/Header/Header.tsx

```diff
@@ -0,0 +1,22 @@
+import { BrandContainer, NavBar, NavMenu, Wrapper } from './styles';
+import { StyledLink, StyledWebName, HomeLink } from './styles';
+import Logo from '../../assets/Logo';
+
+export default function Header() {
+    return(
+        <NavBar>
+            <Wrapper>
+                <BrandContainer>
+                    <HomeLink to='/random'>
+                        <Logo widthProp={60} heightProp={60} />
+                        <StyledWebName>DogLover</StyledWebName>
+                    </HomeLink>
+                </BrandContainer>
+                <NavMenu>
+                    <StyledLink to='/random'>Random dog image</StyledLink>
```

Comment:
When we are on the home route this link not working. What is the correct action here ?

Reply:
I think the best thing to do is removing the random route and leaving only the home route. Then both the logo and the "random dog image" links should navigate to home.

`        <Route path="/" element={<Random />} />
        <Route path="breeds" element={<Breeds />} />
        <Route path="breeds/:breed" element={<BreedGallery />} />
        <Route path="*" element={<NoMatch />} />`
        
        
`<BrandContainer>
                    <HomeLink to='/'>
                        <Logo widthProp={60} heightProp={60} />
                        <StyledWebName>DogLover</StyledWebName>
                    </HomeLink>
                </BrandContainer>
                <NavMenu>
                    <StyledLink to='/'>Random dog image</StyledLink>
                    <StyledLink to='/breeds'>Dog breeds</StyledLink>
                </NavMenu>`